package.json

```diff
@@ -1,90 +1,126 @@
 {
-  "name": "easy-extensibility",
```

Comment:
Did anything actually change here, or is it just linting/formatting?
- If it's the latter, then please revert the change ---i.e., don't commit this. Thanks.

Reply:
This should gone once the .vscode folder is included in the .gitignore file.